VogtPizza/PizzaManager.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPizza
+{
+    class PizzaManager
```

Comment:
Change this to `PizzaBuilder`. That is more accurately describing what this is doing. Given a pizza, it has the logic to "build" it.

Just FYI, usually a "Manager" class can exist without the specific thing it manipulates (pizza in this case) from being a constructor parameter. For example, a pizzaManager would be used for 200 pizzas. You wouldnt create a PizzaManager PER pizza. This is a PizzaBuilder!